### ISSUE TITLE
Add support to Scrollbar to make trail and bar always shown

### ISSUE THREAD

**Issue by msoham123** [flutter/flutter#75075]
For web, it's nice to have the scrollbar on the side to scroll through large amounts of content. 

Scrollbar Code:
```
Scrollbar(
            hoverThickness: 15.0,
            radius: Radius.zero,
            showTrackOnHover: true,
            thickness: 15.0,
            isAlwaysShown: true,
            controller: _scrollController,
            child: ...
),
```

Default behavior while not hovered over:

![image](https://user-images.githubusercontent.com/51520568/106368988-bf917f00-6302-11eb-839c-fa032db6b9b3.png)

Behavior that reflects the web platform: (Only happens in Flutter when hovering over the bar)

![image](https://user-images.githubusercontent.com/51520568/106368962-8527e200-6302-11eb-8cb8-e1060cba0738.png)

Web Scrollbar on Flutter.dev:

![image](https://user-images.githubusercontent.com/51520568/106368980-ae487280-6302-11eb-9d3c-a3573ba5bce5.png)

Is it possible to add a boolean property to the Scrollbar class to enable the trail and deeper scrollbar color all of the time?

**Comment by pedromassangocode** [comment]
Hi @msoham123 
Sorry I'm a bit confused with the issue description. Have you tried `isAlwaysShown` property?
Can you please describe your proposal better? What exactly do you want to achieve?
Thank you

**Comment by msoham123** [comment]
@pedromassango No problem. I should have been a little more specific. If you take a look at the image that shows the standard scrollbar (the one on flutter.dev), you can see that the path that the scrollbar takes is greyed out a bit. However, in flutter, this greyed out behavior only happens when you hover over the scrollbar. I was making a request to add a parameter to keep this greyed out behavior active at all times in order to better reflect the behavior seen with a conventional website.

**Comment by Piinks** [comment]
Hi @msoham123 thanks for filing! Currently, the material/Scrollbar behaves this way in accordance with MaterialDesign. We could expose these properties there, or it might be better to do so on the RawScrollbar. No sure yet. :)

**Comment by msoham123** [comment]
@Piinks Makes sense, considering that the scrollbar was originally implemented for mobile devices. Is this an easy change to make? I could try writing a PR myself, but I'm not sure I understand exactly how the scrollbar works in Flutter.
